DeAceroDLabs/marketplace-frontend
# fix: [DMP-673] Changed Haz to Has.

## What is done

[[DMP-673] Changed Haz to Has.](https://deacero.atlassian.net/browse/DMP-673)

## How is done

1. Quick orthographic change.

## How to test

1. Run the project via `npm start`
2. Go to registrarse.
3. Fill all formats.
4. Successful text should start with Has.
